src/main/java/bio/terra/service/resourcemanagement/google/GoogleResourceDao.java

```diff
@@ -146,22 +147,20 @@ public boolean unlockBucket(String bucketName, String flightId) {
      * Fetch an existing bucket_resource metadata row using the name amd project id.
      * This method expects that there is exactly one row matching the provided name and project id.
      * @param bucketRequest
-     * @return a reference to the bucket as a POJO GoogleBucketResource
+     * @return a reference to the bucket as a POJO GoogleBucketResource or null if not found
      * @throws GoogleResourceException if there is not exactly one matching row
```

Comment:
I think this is no longer true. Now the method returns null if the row is not found, instead of throwing a GoogleResourceNotFoundException. And it throws a CorrupteMetadataException if there are >1 matching rows.

Reply:
OK, I updated the comment and I think correctly described the two exceptions that can be thrown.